float64 x
float64 y
float64 v_x
float64 v_y
float64 psi
float64 psi_dot